xbmc/cores/DllLoader/exports/emu_msvcrt.cpp

```diff
@@ -1633,7 +1631,7 @@ extern "C"
     int ret;
 
     ret = dll_fgetpos64(stream, &tmpPos);
-#if !defined(TARGET_POSIX) || defined(TARGET_DARWIN) || defined(TARGET_FREEBSD) || defined(TARGET_ANDROID)
+#if !defined(__GLIBC__) || defined(TARGET_DARWIN) || defined(TARGET_FREEBSD) || defined(TARGET_ANDROID)
```

Comment:
does this mean that musl is not posix? Or that the else branch here is actually posix instead?